### ISSUE TITLE
[Accessbility]A11y_Java Tooling for IntelliJ/Eclipse - New Features_Application Graph_Screen Reader: Jaws is not announcing any information about the controls in 'Application Graph' Section

### ISSUE THREAD

**Issue by InduPriya1805** [microsoft/azure-tools-for-java#3632]
GitHub Tags: #A11y_JavaToolingforIntelliJ/Eclipse–NewFeature; #A11yMAS; #A11yTCS; #C+E-H; #DesktopApp; #Win10; #JavatoolingforIntelliJ/Eclipse; #ScreenReader; #JAWS; #MAS1.3.1; 

[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier

**Environment Details:**
Download and Install IntelliJ IDEA from https://www.jetbrains.com/idea/download/#section=windows
IntelliJ IDEA 2019.2.1
Build #IU-192.6262.58

**OS Details:**
OS: Windows (Version 1903, Build: 18362.113)

**MAS Violated:** 1.3.1

**Note:
Jaws is not reading any information present in the jobs section**

**Repro Steps:**
1)Turn on screen reader (Jaws) 
2)Launch IntelliJ IDEA
3)Tab till Create New Project button and press enter. Navigate to Azure Spark/HDInsight-->Spark Project Scala and press Next button and fill all the details to create a new project
4)Tab till Azure Spark/HDInsight-->Spark Project Scala
5)Tab till Build Tools-->Maven
6)Tab till Next button and press enter
7)Navigate to project name-->src-->main-->scala and use keyboard shortcut key to open the context menu
8)Navigate to New-->Scala Class and press enter
9)Tab till object from menu and select it using keyboard
10)Override the code and Navigate to project name-->src-->main-->scala-->Project Name and right click on it & Tab till Submit Spark Application and press enter
11)Fill all the required details & Tab till finish button and press enter
12)Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription
13)Tab till HDInsight and link a cluster. Navigate to jobs under the cluster and press enter
14)Select any one the application and press enter
15)Verify whether screen reader is reading the information about the controls in the 'Application Graph' section or not

**Actual Result:**
Jaws is not announcing any information about the controls in the 'Application Graph' section when keyboard focus is going to the controls

**Expected Result:**
Jaws should announce the information about the controls in the 'Application Graph' section when navigating to the controls using 'Tab' key

**User Impact:**
Screen Reader users will not understand about the controls where keyboard focus is going, if it doesn't announce any information

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment:**
[Bug_3632_jaws is not reading tab names and links.pptx](https://github.com/microsoft/azure-tools-for-java/files/3655983/Bug_3632_jaws.is.not.reading.tab.names.and.links.pptx)

**Comment by t-rufang** [comment]
The job view page is wrote with the help of JavaFX which has no support for Accessibility. Please refer to JDK issue [JDK-8177382](https://bugs.openjdk.java.net/browse/JDK-8177382)

**Comment by Alexandria-R** [comment]
This has been approved to close by SME.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749728
